examples/readme.rs

```diff
@@ -0,0 +1,60 @@
+use actix_web::{get, web, App, HttpResponse, HttpServer};
+use biscuit_actix_middleware::BiscuitMiddleware;
+use biscuit_auth::{macros::*, Biscuit, PublicKey};
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    println!(
+        r#"
+This server exposes a single endpoint on `GET /hello`.
+The whole server requires a valid biscuit token. Its signature is verified
+by a public key provided through the `BISCUIT_PUBLIC_KEY` environment variable.
+The `GET /hello` endpoints expects a token containg `role("admin");`.
+
+You can generate a keypair and a token with the biscuit CLI:
+> biscuit keypair # generates a key pair
+> echo 'role("admin");' | biscuit generate --private-key <the private key> -
+
+You can also use the following:
+
+  public key: 2d6a07768e5768192870f91a6949cd09ce49865f2e2eb1241369c300ee7cc21f
+  token: EnYKDBgDIggKBggGEgIYDRIkCAASIDZy3NpVVceLLr5Xqcv08H7BeBQry38djs13jJz6uDxVGkBWQyFbDLPYaEo1PMZxB6In0mbYFiAjWEJfd2kr7P2qu8YQDNCoyIBsRP4A-4OzfvzFr2o3x9b7jOHksiRxbpILIiIKIITNLF9dFYE_tbpsqBEgno0bbwLi56dvpM43SaK7o8Iu
+
+The token has to be set in the authorization header:
+
+  curl -v http://localhost:8080/hello \
+    -H "Authorization: Bearer <token>"
+"#
```

Comment:
Is fmt responsible for 0 indent?

Reply:
this is a multiline string, so leaving indents in would also indent the lines in the output.

Reply:
there are external macros for that, but i don't think it's worth depending on them here.